lib/tasks/migrate.rake

```diff
@@ -117,6 +117,11 @@ namespace :migrate do
         .update_all(lifecycled_at: Time.current.to_s(:db))
       end
     end
+
+    desc "Fill live_project"
+    task fill_live_project: :environment do
+      Classification.where(live_project: nil).update_all("live_project = (metadata->>'live_project')::boolean")
```

Comment:
split it into small batches and update them, ala what you did [here](https://github.com/zooniverse/Panoptes/blob/04d6d77eed29b2e9ced5ff771707851bbb679f11/lib/tasks/migrate.rake#L80)